src/Renderer/ThreeExtended/Feature2Mesh.js

```diff
@@ -115,6 +115,11 @@ function addExtrudedPolygonSideFaces(indices, length, ring, isClockWise) {
             indices.push(i + 1);
         }
     }
+
+    // last indexes are too high - wrap them inside boundaries
```

Comment:
This is ugly :)

Maybe use the modulo in the previous loop, for all `i + ...` cases.
Something like: `mod = ring.offset + ring.count + length;`
Then:
```js
...
indices.push((i + length) % mod);
indices.push(i);
indices.push((i + length + 1) % mod);
...
```

Reply:
Yes, but if we do the modulo operation each time on a big ring, we compute it for nothing. But I guess we won't have complex shapes, so I'm okay with changing it.